tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -992,10 +1005,62 @@ void SPIRVEmitter::doReturnStmt(const ReturnStmt *stmt) {
   }
 }
 
+bool SPIRVEmitter::isBreakStmtLastStmtInCaseStmt(const BreakStmt *breakStmt,
```

Comment:
Cool stuff!

Minor: It's more readable for me if `breakStmtIsLastStmtInCaseStmt`. But I don't care that much.